cross/Dockerfile

```diff
@@ -88,6 +88,14 @@ RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add - && \
     DEBIAN_FRONTEND=noninteractive apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && \
     rm -f /etc/apt/sources.list.d/yarn.list
 
+# ------------------------------------------------------------------------------------------------
+# install kind
+RUN curl -Lo /usr/local/bin/kind https://github.com/kubernetes-sigs/kind/releases/download/v0.4.0/kind-linux-amd64 -s &&\
```

Comment:
If these versions (0.4.0, 1.15.0) are specified as `ARGS`, then pull requests in crossplane (and provider) projects can bump their build requirements to newer versions as needed without requiring changes in this project (and coordination among subordinate projects).

Reply:
The trade off would be less code readability as the versions and the wiring will be kept away from the Dockerfile. How often do you think these versions will be bumped? 

As a separate concern is imagine we'd use the same version of these tools across multiple projects, and going them in one place would be more manageable.

Reply:
The default values can be provided: https://docs.docker.com/engine/reference/builder/#default-values so the value would be centralized (this repository) but we could drift away from that default in a provider by provider  / branch by branch case.

Reply:
Seem's like we are not putting tools in the image anyway. Reverting the change in this Dockerfile, and putting tools targets in Crossplane Makefile.